README.md

```diff
@@ -8,92 +8,145 @@
+   ```bash
+   ngrok http 3000
+   ```
 
-1. Once you have started ngrok, update your TwiML app's voice URL setting to use your ngrok hostname, so it will look something like this:
+6. Once you have started ngrok, update your [TwiML app's](#create-a-twiml-app) Voice URL setting to use
+   your ngrok hostname, so it will look something like this: 
+   
+   ```
+   http://88b37ada.ngrok.io/call/connect
+   ```
+   
+   See [Create a TwiML App](#create-a-twiml-App).
 
-    `http://88b37ada.ngrok.io/call/connect`
+7. Everything is setup, now you can open two tabs in the browser to test the application.
 
-### Try it out
-1. To create a support ticket go to the `home` page. On this page you could fill some fields and create a ticket or you can call to support:
+   - To create a support ticket go to the `home` page. On this page you could fill some fields and create a ticket or you can call to support:
 
-    [http://localhost:3000](http://localhost:3000)
+       [http://localhost:3000](http://localhost:3000)
 
-1. To respond to support tickets go to the `dashboard` page (you should open two windows or tabs). On this page you could call customers and answers phone calls.
+   - To respond to support tickets go to the `dashboard` page (you should open two windows or tabs). On this page you could call customers and answers phone calls.
 
-    [http://localhost:3000/dashboard](http://localhost:3000/dashboard)
+       [http://localhost:3000/dashboard](http://localhost:3000/dashboard)
 
-## Run the tests
+That's it!
+
+### Tests
 
 You can run the tests locally by typing
 
 ```bash
 npm test
 ```
 
-## Meta
+### Cloud deployment
+
+Additionally to trying out this application locally, you can deploy it to a variety of host services. Here is a small selection of them.
+
+Please be aware that some of these might charge you for the usage or might make the source code for this application visible to the public. When in doubt research the respective hosting service first.
+
+| Service                           |                                                                                                                                                                                                                           |
+| :-------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [Heroku](https://www.heroku.com/) | [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)                                                                                                                                       |
```

Comment:
I don't think Heroku works for this one unless you create an `app.json` file that configures Heroku to also spin up a MongoDB database. So feel free to remove this for now.